### DISCUSSION TITLE
v-data-table not showing items

### DISCUSSION THREAD

**Discussion by susanwands** [vuetifyjs/vuetify#14381] [Q&A]
I am using v-data-table with server side filtering, paging and sorting. The table seems to be rendering fine until I navigate away from the page and come back. Then the table says there is no data available. 

- I can render the data on the page in a separate div. 
- I can see the data in my Vue console. 
- I can see the graphql call on my network tab. 

For some reason, the data is not showing in the table. 

- If I refresh the page the data is shown in the table again. 
- If I enter a search value that forces a graphql call and refreshes the array bound to the :item property, the data is shown in the table again. 

How do I get the v-data-table to see the data values that are there without having to refresh them?

**Comment by nekosaur** [comment]
You would have to show your code for anyone to be able to help here.
